docs/topics/ksp/ksp-cmdline.md

```diff
@@ -0,0 +1,62 @@
+[//]: # (title: Running KSP from command line)
```

Comment:
Also, there is no need to short the name (SEO purposes).
ksp-command-line looks fine to me